### PR TITLE
Updated tutorial - included terraform and updated query

### PR DESCRIPTION
**Pull request by aburhan**

### CODE REVIEW

File: .github/workflows/gke-vpa-recommendations-ci.yml [hunk 1]
```diff
@@ -15,11 +15,8 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v3
-      - name: build metric-exporter container
```
Comment (bourgeoisor) [inline .github/workflows/gke-vpa-recommendations-ci.yml:18]: This container still exist no? Should the "validate terraform" override this step, or create a new one?

Comment (aburhan) [reply .github/workflows/gke-vpa-recommendations-ci.yml]: @bourgeoisor Thank you for catching this. Yes, I still need this container 'metric-exporter'.

removed hpa-metrics-exporter.This container is no longer needed.

File: .github/workflows/gke-vpa-recommendations-ci.yml [hunk 2]
```diff
@@ -19,7 +19,8 @@ jobs:
         run: |
           cd gke-vpa-recommendations/metrics-exporter
           docker build --tag metric-exporter .
-      - name: build hpa-metrics-exporter container
+      - name: Validate terraform
         run: |
-          cd gke-vpa-recommendations/scripts
```
Comment (bourgeoisor) [inline .github/workflows/gke-vpa-recommendations-ci.yml:24]: Is that image not used for the tutorial anymore? If so perhaps we should delete it (and relevant obsolete files) from here?

https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/58208b054815740b4fc8b53f11053e0b162d7a94/gke-vpa-recommendations/scripts/

File: gke-vpa-recommendations/terraform/cloudrun.tf [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        }
+        env {
+            name = "LOGGING_LEVEL"
+            value = "INFO"
+        }
+        env {
+            name = "SERVICE_ACCOUNT"
+            value = "default"
+        }
+        env {
```
Comment (cliles) [inline gke-vpa-recommendations/terraform/cloudrun.tf:42]: Where are these env vars used by the metrics-exporter?

File: .github/workflows/gke-vpa-recommendations-ci.yml (hunk 2 above)
Comment (aburhan) [reply .github/workflows/gke-vpa-recommendations-ci.yml:24]: no its no longer used in the tutorial

File: gke-vpa-recommendations/terraform/cloudrun.tf (hunk 3 above)
Comment (aburhan) [reply gke-vpa-recommendations/terraform/cloudrun.tf]: @cliles this was a left over variable for the original version of the code. I've cleaned up the terraform and updating the guide.

File: .github/workflows/gke-vpa-recommendations-ci.yml (hunk 2 above)
Comment (aburhan) [reply .github/workflows/gke-vpa-recommendations-ci.yml:24]: also removed the files supporting the hpa metric exporter

**Review by bourgeoisor — APPROVED** [review]
LGTM

**Pull request merged**